docs/ovms_quickstart.md

```diff
@@ -0,0 +1,170 @@
+# OpenVINO&trade; Model Server Quickstart
+
+## Deploying Model Server
+
+Model Server can deployed using any of the following options:
+1. Download a pre-built container image from Docker Hub
+2. Build a container image with a Dockerfile
+3. Build the application locally from source 
+
+Each approach is described below.
+
+### Pre-built Docker Container
+
+A pre-built container with OpenVINO&trade; Model Server is available for download on [Docker Hub](https://hub.docker.com/r/openvino/ubuntu18_model_server). The image
+is based on Ubuntu 18.04 and contains all required components needed to perform inference. 
+
+To start using this image, you can pull it with the following command:
+
+```bash
+docker pull openvino/ubuntu18_model_server
+```
+
+### Build with a Dockerfile
+
+Three Dockerfiles are provided that can be used to build a container image locally. There is also a Makefile to simplify building an image. The Dockerfiles are listed below:
+
+#### 1. [Ubuntu 18 Dockerfile](https://github.com/openvinotoolkit/model_server/blob/master/Dockerfile)
+
+To build this image use following command from the `model_server` directory:
+
+```bash
+make docker_build_apt_ubuntu
+```
+
+#### 2. [Clear Linux Dockerfile](https://github.com/openvinotoolkit/model_server/blob/master/Dockerfile_clearlinux)
+
+To build this image use following command from the `model_server` directory:
+
+```bash
+make docker_build_clearlinux
+```
+
+#### 3. [Based on OpenVINO Toolkit](https://github.com/openvinotoolkit/model_server/blob/master/Dockerfile_binary_openvino)
+
+In this case an image is built based on OpenVINO Toolkit - to build it you need to obtaing an URL to the toolkit package. It can be done by registering on the [OpenVINO&trade; Toolkit website](https://software.intel.com/en-us/openvino-toolkit/choose-download).
+
+When you have this URL, please issue the following command - passing there just obtained URL:
+
+```bash
+make docker_build_bin dldt_package_url=<url-to-openvino-package-after-registration>/l_openvino_toolkit_p_2020.1.023_online.tgz
+```
+
+### Build Locally from Source
+
+Instructions to build locally are [here](https://github.com/openvinotoolkit/model_server/blob/master/docs/host.md).
+
+
+## Running Model Server

```

Comment:
I think we are duplicating here a lot of content from docker_container.md
Maybe it would be better in include here examples how to start OVMS with public models and use existing client?
Easiest deployment example should be with public GS models.

Reply:
I agree that we should include example with public models. Which do you suggest @dtrawins? 

What do you want removed? Should we just link some sections to the docker_container.md page?

Reply:
here is a public model with int8 precision https://storage.googleapis.com/public-artifacts/intelai_public_models/resnet_50_i8/1/resnet_50_i8.xml and https://storage.googleapis.com/public-artifacts/intelai_public_models/resnet_50_i8/1/resnet_50_i8.bin. It could be accessed also via gs://public-artifacts/intelai_public_models/resnet_50_i8. There are also models from model zoo https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/3/
Maybe in the quick state they should be describe basic path with a link to complete doc or more advanced cases?

Reply:
I modified location of the model zoo.